deployment/terraform/cluster.tf

```diff
@@ -143,6 +143,38 @@ resource "aws_rds_cluster" "db_cluster" {
   vpc_security_group_ids = ["${aws_security_group.db.id}"]
 }
 
+resource "aws_instance" "loadtest_agent" {
+  tags = {
+    Name = "${var.cluster_name}-agent-${count.index}"
+  }
+
+  connection {
+    type = "ssh"
+    user = "ubuntu"
+    host = self.public_ip
+  }
+
+  ami           = "ami-0fc20dd1da406780b"
+  instance_type = "t2.medium"
+  key_name      = aws_key_pair.key.id
+  count         = var.loadtest_agent_count
+
+  vpc_security_group_ids = ["${aws_security_group.agent.id}"]
+
+  provisioner "remote-exec" {
+      inline = [
+      "while [ ! -f /var/lib/cloud/instance/boot-finished ]; do echo 'Waiting for cloud-init...'; sleep 1; done",
+      "wget https://dl.google.com/go/go${var.go_version}.linux-amd64.tar.gz",
+      "sudo tar -C /usr/local -xzf go${var.go_version}.linux-amd64.tar.gz",
+      "sudo sh -c \"echo 'export PATH=$PATH:/usr/local/go/bin' >> /etc/profile\"",
+      "wget https://github.com/mattermost/mattermost-load-test-ng/archive/${var.loadtest_source_code_ref}.tar.gz",
+      "tar -xzf ${var.loadtest_source_code_ref}.tar.gz",
+      "cp mattermost-load-test-ng-${var.loadtest_source_code_ref}/config/simplecontroller.default.json mattermost-load-test-ng-${var.loadtest_source_code_ref}/config/simplecontroller.json",
+      "cp mattermost-load-test-ng-${var.loadtest_source_code_ref}/config/config.default.json mattermost-load-test-ng-${var.loadtest_source_code_ref}/config/config.json",
```

Comment:
We are already overwriting this later. Do we still need to do this ?

Reply:
We are overwriting for the coordinator only, but we start normal agents before we configure coordinator. And the need for this will be vanished after I send the fix 👍